### ISSUE TITLE
Fails to Download from HITOMI

### ISSUE THREAD

**Issue by wshsirin8** [avluis/Hentoid#1206]
**Hentoid Version**: Hentoid v1.20.7 (761)

**Device/Android Version**: Redmi K70 Pro / Android 14

**Issue details / Repro steps**: I am able to access HITOMI normally, but all comics fail to download.

**Stack trace / LogCat**:
```ruby
The latest log information shows deletion entries caused by adding failed downloads back to the download queue.
```
**Additional Details**: I have tested the issue on another Android device and different network environments, but I still cannot download any comics. Perhaps this is the same as #1203?

![Screenshot_2024-12-21-11-36-50-983_me devsaki hentoid](https://github.com/user-attachments/assets/aaaa08ef-1a08-4e83-846c-c609867a19b5)

@AVnetWS/admin-team

**Comment by Pantoprazol** [comment]
This has already been adressed at the Discord, its a Serverside problem of Hitomi and they are working on fixing it.

**Comment by wshsirin8** [comment]
Thank you for the update! I understand, I'll wait for them to fix the issue. Please let me know if there are any updates. Thanks!

**Comment by RobbWatershed** [comment]
Fixed in today's v1.20.8
